graphics/blit/blit-scale.cpp

```diff
@@ -41,14 +41,17 @@ static void scaleVertical(byte *dst, const byte *src,
                           const byte flip, const uint bytesPerPixel) {
 	const bool flipy = flip & FLIP_V;
 
+	// 16.16 fixed point
+	const uint srcIncY = (srcH << 16) / dstH;
```

Comment:
Please specify uint32 everywhere as you are expecting a 32-bits variable.

Reply:
OK, done.